tests/PHPStan/Rules/Comparison/StrictComparisonOfDifferentTypesRuleTest.php

```diff
@@ -1001,4 +1001,9 @@ public function testHashing(): void
 		]);
 	}
 
+	public function testBug12772(): void
```

Comment:
Please add also `$_ENV['CI']` test explicitly, as I want to make sure GitHub CI environment variable is never assumed to exist on runtime - https://github.com/atk4/ui/blob/235416b31066827226e462f1916d9fc99c0d14b4/demos/init-db.php#L556.

Reply:
I get your point, but adaptions here are not related to any environment variables. If Ondřej is fine with this, I expect it to be merged very soon and then we should also see the effects